### PR TITLE
SUR-228 Dropzone

### PR DESCRIPTION
**Pull request by BabuTechBsf**
### Description

Added Dropzone component

### Screenshots

https://d.pr/v/SF3oYt

### Checklist:

-   [ ] My code is tested
-   [x] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/dropzone/dropzone.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	parameters: {
+		layout: 'centered',
+	},
+	tags: [ 'autodocs' ],
+	argTypes: {
+		size: {
+			control: { type: 'select' },
+			options: [ 'sm', 'md', 'lg' ],
+			description: 'Size of the dropzone component',
+			defaultValue: 'sm',
```
Comment (jaieds) [inline src/components/dropzone/dropzone.stories.tsx:18]: We should not define these manually.
```suggestion
```

**Review by jaieds — CHANGES_REQUESTED** [review]
- Add transition on hover.
- Fix gap and alignment. Screenshot: https://d.pr/i/Ky0zKA
- Fix the loader background. Screenshot: https://d.pr/i/o3Ng5F
- Focus state border is not present.
- Showing infinite loader. Screencast: https://d.pr/v/KAtPTq

File: src/components/dropzone/dropzone.stories.tsx (hunk 1 above)
Comment (BabuTechBsf) [reply src/components/dropzone/dropzone.stories.tsx:18]: My bad. Resolved.

**Review by JelenaTakac — CHANGES_REQUESTED** [review]
Please check video: https://bsf.d.pr/v/NYPjEe
In video mentioned some suggestions for improvements.

**Comment by BabuTechBsf** [comment]
> * Add transition on hover.
> * Fix gap and alignment. Screenshot: https://d.pr/i/Ky0zKA
> * Fix the loader background. Screenshot: https://d.pr/i/o3Ng5F
> * Focus state border is not present.
> * Showing infinite loader. Screencast: https://d.pr/v/KAtPTq

Transition added.
Gaps is not given, Alignment adjusted.
background removed from component level.
focus state added.
loading fixed.

**Comment by BabuTechBsf** [comment]
> Please check video: https://bsf.d.pr/v/NYPjEe In video mentioned some suggestions for improvements.

As per discussion with sujay on previous SRM. 
At this version we are not providing multi selection. 
Also loader was mentioned to be fake loader. So it's not a real loader of progress.
other issues fixed.

File: src/components/dropzone/dropzone.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// Dropzone.stories.js
+
+import Dropzone from './dropzone'; // Adjust the import path as needed
+import { Meta, StoryFn } from '@storybook/react';
```
Comment (jaieds) [inline src/components/dropzone/dropzone.stories.tsx:4]:
```suggestion
import type { Meta, StoryFn } from '@storybook/react';
```

**Pull request merged**